src/console/Microsoft.Health.Fhir.Ingest.Console.Normalization/Microsoft.Health.Fhir.Ingest.Console.Normalization.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<LangVersion>10.0</LangVersion>
+	</PropertyGroup>
+	<PropertyGroup>
+		<IsPackable>true</IsPackable>
+	</PropertyGroup>
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.0" />
+		<PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.2.0">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+	</ItemGroup>
+
+	<ItemGroup>
+		<ProjectReference Include="..\Microsoft.Health.Fhir.Ingest.Console.Common\Microsoft.Health.Fhir.Ingest.Console.Common.csproj" />
+		<ProjectReference Include="..\..\lib\Microsoft.Health.Common\Microsoft.Health.Common.csproj" />
+		<ProjectReference Include="..\..\lib\Microsoft.Health.Events\Microsoft.Health.Events.csproj" />
+		<ProjectReference Include="..\..\lib\Microsoft.Health.Fhir.Ingest\Microsoft.Health.Fhir.Ingest.csproj" />
+		<ProjectReference Include="..\..\lib\Microsoft.Health.Fhir.R4.Ingest\Microsoft.Health.Fhir.R4.Ingest.csproj" />
+		<ProjectReference Include="..\..\lib\Microsoft.Health.Expressions\Microsoft.Health.Expressions.csproj" />
+		<PackageReference Include="Microsoft.Health.Fhir.R4.Client" Version="2.0.55" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<None Update="local.appsettings.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+			<CopyToPublishDirectory>Never</CopyToPublishDirectory>
+		</None>
+	</ItemGroup>
```

Comment:
Do we need to commit this ItemGroup definition? This sounds like a file that would need be created by a local dev/end user.

Reply:
we don't need to commit the item group but I do think the local.appsettings.json file helps with testing. updated the PR to have the item group not copy the file by default so the user should not have to take any action. If the user wants to use the local.appsettings.json file to configure settings for testing then they can change the behavior of the item group to copy the file to the output directory.